desktop/apps/auction2/components/range_slider/rc_slider.styl

```diff
@@ -0,0 +1,290 @@
+.rc-slider {
```

Comment:
This is horrendous... I couldn't figure out how to import the css file from this node module, so I copied it here (just for now).